app/controllers/nonsipp/receivetransfer/TransfersInCYAController.scala

```diff
@@ -55,51 +58,82 @@ class TransfersInCYAController @Inject()(
 
   def onPageLoad(srn: Srn, index: Max300, mode: Mode): Action[AnyContent] =
     identifyAndRequireData(srn) { implicit request =>
-      (
-        for {
-          memberDetails <- request.userAnswers.get(MemberDetailsPage(srn, index)).getOrRecoverJourney
-          secondaryIndexes <- request.userAnswers
-            .get(TransfersInSectionCompletedForMember(srn, index))
-            .map(_.keys.toList.flatMap(refineStringIndex[Max5.Refined]))
-            .getOrRecoverJourney
-        } yield {
-          secondaryIndexes
-            .traverse(
-              journeyIndex =>
-                for {
-                  schemeName <- request.userAnswers
-                    .get(TransferringSchemeNamePage(srn, index, journeyIndex))
-                    .getOrRecoverJourney
-                  transferType <- request.userAnswers
-                    .get(TransferringSchemeTypePage(srn, index, journeyIndex))
-                    .getOrRecoverJourney
-                  totalValue <- request.userAnswers
-                    .get(TotalValueTransferPage(srn, index, journeyIndex))
-                    .getOrRecoverJourney
-                  transferReceived <- request.userAnswers
-                    .get(WhenWasTransferReceivedPage(srn, index, journeyIndex))
-                    .getOrRecoverJourney
-                  transferIncludeAsset <- request.userAnswers
-                    .get(DidTransferIncludeAssetPage(srn, index, journeyIndex))
-                    .getOrRecoverJourney
-                } yield TransfersInCYA(
-                  journeyIndex,
-                  schemeName,
-                  transferType,
-                  totalValue,
-                  transferReceived,
-                  transferIncludeAsset
-                )
-            )
-            .map {
-              case Nil => Redirect(controllers.routes.JourneyRecoveryController.onPageLoad())
-              case journeys => Ok(view(viewModel(srn, memberDetails.fullName, index, journeys, mode)))
-            }
-            .merge
-        }
-      ).merge
+      onPageLoadCommon(srn: Srn, index: Max300, mode: Mode)(implicitly)
```

Comment:
Just spotted this, it was a mistake in the original example I think. Should this be: 
```
onPageLoadCommon(srn, index, mode)(implicitly)
```
? (same in onPageLoadViewOnly)

Reply:
If you agree, could you please change it in basic / financial details too @narendrapaduchuri ?

Reply:
Yes, @kberec I agree with you. I changed across above mentioned and  member contributions as well. Thanks

Reply:
Thanks. Just the commits needs squashed and it is good to get merged.